cmd/utils/nodecmd/genesis_test.go

```diff
@@ -33,50 +33,6 @@ var customGenesisTests = []struct {
 	result  []string
 }{
 	// Plain genesis file without anything extra
-	{
-		genesis: `{
-			"alloc"      : {},
-			"blockScore" : "0x20000",
-			"extraData"  : "",
-			"gasLimit"   : "0x2fefd8",
-			"parentHash" : "0x0000000000000000000000000000000000000000000000000000000000000000",
-			"timestamp"  : "0x00"
-		}`,
-		query:  []string{"klay.getBlock(0).parentHash"},
-		result: []string{"0x0000000000000000000000000000000000000000000000000000000000000000"},
-	},
-	// Genesis file with an empty chain configuration (ensure missing fields work)
-	{
-		genesis: `{
-			"alloc"      : {},
-			"blockScore" : "0x20000",
-			"extraData"  : "",
-			"gasLimit"   : "0x2fefd8",
-			"parentHash" : "0x0000000000000000000000000000000000000000000000000000000000000000",
-			"timestamp"  : "0x00",
-			"config"     : {}
-		}`,
-		query:  []string{"klay.getBlock(0).parentHash"},
-		result: []string{"0x0000000000000000000000000000000000000000000000000000000000000000"},
-	},
-	// Genesis file with specific chain configurations
-	{
-		genesis: `{
-			"alloc"      : {},
-			"blockScore" : "0x20000",
-			"extraData"  : "",
-			"gasLimit"   : "0x2fefd8",
-			"parentHash" : "0x0000000000000000000000000000000000000000000000000000000000000000",
-			"timestamp"  : "0x00",
-			"config"     : {
-				"homesteadBlock" : 314,
-				"daoForkBlock"   : 141,
-				"daoForkSupport" : true
-			}
-		}`,
-		query:  []string{"klay.getBlock(0).parentHash"},
-		result: []string{"0x0000000000000000000000000000000000000000000000000000000000000000"},
-	},
 	{
```

Comment:
Minor comment:
It will be nice if we add other genesis cases such as round-robin in future PR.

Reply:
round-robin test case has been added. please take a look :)